song-server/src/main/java/bio/overture/song/server/controller/SubmitController.java

```diff
@@ -55,7 +50,8 @@ public class SubmitController {
   public SubmitResponse submit(
       @RequestHeader(value = AUTHORIZATION, required = false) final String accessToken,
       @PathVariable("studyId") String studyId,
+      @RequestParam(value = "ignoreDuplicates", defaultValue = "false") boolean ignoreDuplicates,
```

Comment:
ignoreDuplicates set to false by default

Reply:
for my own clarity, "ignore" means allow? or does it skip them?

Reply:
**ignoreDuplicates** as true will "allow" to submit duplicate analysis.